src/main/java/frc/trigon/robot/subsystems/swerve/Swerve.java

```diff
@@ -301,30 +290,35 @@ private ChassisSpeeds powersToSpeeds(double xPower, double yPower, double thetaP
         );
     }
 
-    private ChassisSpeeds calculateSelfRelativePIDToPoseSpeeds(FlippablePose2d targetPose) {
-        final Pose2d currentPose = RobotContainer.ROBOT_POSE_ESTIMATOR.getPredictedRobotPose(0.13);
+    private ChassisSpeeds calculateTargetSelfRelativeSpeedsFromPose(FlippablePose2d targetPose) {
```

Comment:
Again, toPose is more correct, and now this has no mention of PID making it less clear what this does.